### PR TITLE
Add testplan

### LINKED ISSUE #895: Create a manual test plan and turn it into automated tests
We should develop a comprehensive manual test plan and turn it into automated tests. Here's what I have so far:

* Draw every shape, multi-segment arrow, multi-segment line, single-line text, and multi-line text on the canvas
* Change the properties (color etc) of a shape and text
* Resize an element (try every resize handle)
* Click select an element and drag it.
* Alt-drag to duplicate an element
* Click an an element, shift-click others to select a group. Drag the group.
* Pinch-to-zoom works on both desktop touchpad and iOS
* 2-finger-scroll works on desktop touchpad
* Export the file. The preview looks correct. Download the file
* Click the trash button. It clears the screen.
* Reload the file saved above. It looks correct.
* Load a [big file](https://drive.google.com/open?id=1Yb4UPBiHVXQMmqMN73tirl2zHkKlabE3) and drag one of the elements. It drags smoothly.

### PR DESCRIPTION
**Pull request by petehunt**
Closes #895. We can add automated tests later. Let's at least document a comprehensive manual test plan for contributors to run before landing a PR.

### CODE REVIEW

**Comment by Fausto95** [comment]
Worth adding this to the PR template so that a PR is ready to be to reviewed only if every steps are marked as done?

**Comment by petehunt** [comment]
force pushed this to my fork's master branch and opened a [test PR](https://github.com/petehunt/excalidraw/pull/1), it seems to work

**Comment by Fausto95** [comment]
We're good to go

**Comment by dwelle** [comment]
:+1: 

Though I don't think we should necessarily require to go through all that for every PR, because that's pretty heavy.

**Comment by Fausto95** [comment]
We could at least consider checking the most critical ones

File: .eslintignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules/
 build/
 package-lock.json
 .vscode/
+firebase/
```
Comment (lipis) [inline .eslintignore]: End of line

**Comment by lipis** [comment]
Yeah.. I also agree that's it's quite a big list.. will scare some people.. we should automate

**Comment by petehunt** [comment]
it's pretty heavy and maybe over-indexes on test plans for large refactorings.

i can take a swing at updating this to include a minimum viable test plan and an optional longer one

**Comment by petehunt** [comment]
deprecated in favor of #1053

**Pull request closed without merging**